src/components/GlobalPreferences/CustomLabels/useSelectIdentities.js

```diff
@@ -47,6 +47,10 @@ function useSelectIdentities(identities, filteredIdentities) {
     }
   }, [initialSelected, setSelected, firstRender])
 
+  useEffect(() => {
+    setFirstRender(true)
```

Comment:
@AquiGorka it seems that `firstRender` doesn’t refer to a render in the React sense? What would you think of renaming it to something that describes better what it is used for?

Reply:
So, there was some back and forth with this var name before, I don't mind changing it but I would like to avoid having to come back to it in the future ([this](https://github.com/aragon/aragon/pull/909/commits/0ae489666bfffa6991d176b812e989f5a488ba54) is where the name comes from)

Reply:
Maybe somewhere in the line of: `resetSelection`?

Reply:
It was still true at the time (`firstRender` was reflecting the fact that we are in the first render or not), but it’s not true anymore with this PR’s change, since we are setting it to `true` in a render late in the component’s life. I don’t see any problem with changing names to better fit new purposes, quite the contrary IMO.

Reply:
Sure I follow the line of thought, what about my suggestion? Any other name that comes to mind?

Reply:
Oh I didn’t see it sorry, yes that sounds better!